.github/workflows/release-binary.yml

```diff
@@ -0,0 +1,80 @@
+name: Release Binaries
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  build:
+    name: Build and Upload Release Assets
+    runs-on: ubuntu-latest
+    container: golang:1.18-bullseye
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build Linux/amd64 binary
+        run: |
+          export GO111MODULE=on
+          export CGO_ENABLED=0
```

Comment:
Unless I'm missing something GO111MODULE and CGO_ENABLED are both constant for all these options.  You can define these env vars at the top level using an "env" block.  We do something like that in the python-driver-wheel actions [here](https://github.com/datastax/python-driver-wheels/blob/master/.github/workflows/build_wheels_mac.yml#L21-L32).

Reply:
Fixed.